### ISSUE TITLE
Template Grafana dashboard

### ISSUE THREAD

**Issue by MrChips821** [percona/mongodb_exporter#693]
Does anyone have a template / example grafana dashboard that works with the metrics from this exporter they would be willing to share?
I can't see any on the grafana website.

**Comment by abhinavguptaquizizz** [comment]
Following. I am looking for something similar

**Comment by BupycHuk** [comment]
Hi, we have our dashboards in [github.com/percona/grafana-dashboards](https://github.com/percona/grafana-dashboards/tree/main/dashboards/MongoDB) repository and there is an [option to export](https://github.com/percona/grafana-dashboards/tree/main#reusing-dashboards-outside-of-pmm) our dashboards to vanilla grafana.

**Comment by MrChips821** [comment]
Thanks Nurlan. Will take a look at that .

Sent from Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by dirodriguezm** [comment]
Hello @BupycHuk, are these dashboards up to date ? I don't see the metrics used by the dashboards on my instance of mongodb_exporter.

**Comment by BupycHuk** [comment]
@dirodriguezm Yes, you need to enable compatible mode, some dashboards are still use metrics from v1

**Comment by dirodriguezm** [comment]
Ok I'll do that, thanks for answering
